test/main.cpp

```diff
@@ -0,0 +1,10 @@
+/***************************************************************************
+ * Copyright (c) 2023, xeus-cpp contributors
+ *
+ * Distributed under the terms of the BSD 3-Clause License.
+ *
+ * The full license is in the file LICENSE, distributed with this software.
+ ****************************************************************************/
+
+#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
+#include "doctest/doctest.h"
```

Comment:
Why do we need this file? Can we move its contents in `test_interpreter`?

Reply:
Ahh maintaing consistency across all xeus kernels. xeus-kernels like xeus-python & xeus-cling use doctests through a main.cpp file.